install-dependencies.sh

```diff
@@ -121,6 +121,7 @@ fedora_packages=(
     binaryen
     lcov
 
+    lld
```

Comment:
Doesn't seem related.

Reply:
it is related to the drop of `s390x` support, i will do it in a separate commit